src/chirp_client.cc

```diff
@@ -105,10 +107,10 @@ int ChirpClient::follow(const std::string& user,
   }
 }
 
-int ChirpClient::read(const std::string& chirp_id) {
+int ChirpClient::read(const std::string &chirp_id) {
```

Comment:
Minor: I'd recommend having reformatting be a separate pull request.  But it's fine.